src/components/Transfer/Submit.js

```diff
@@ -63,11 +65,15 @@ class Submit extends React.Component {
 					})
 				);
 
+				this.cancel(dispatch);
```

Comment:
`this.refuse()` then :p

And no need of the dispatch !
=> `.catch(() => this.refuse());`